api/src/main/java/openconsensus/stats/Measurement.java

```diff
@@ -18,8 +18,6 @@
 
 import com.google.auto.value.AutoValue;
 import javax.annotation.concurrent.Immutable;
-import openconsensus.stats.Measure.MeasureDouble;
-import openconsensus.stats.Measure.MeasureLong;
 
 /**
  * Immutable representation of a Measurement.
```

Comment:
I like Measure and creating Measurement's from them but it's unclear to me what the utility of this class is. Certainly you need Measurement but the creation of Measurements should be done by Measure.